### PR TITLE
Port the side tabs to prime

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: common/src/main/scala/explore/components/ui/ExploreStyles.scala [hunk 1]
```diff
@@ -69,6 +69,9 @@ object ExploreStyles:
   val MainMenuDropdown: Css      = Css("main-menu-dropdown")
   val SideTabs: Css              = Css("sidetabs")
   val SideButton: Css            = Css("side-button")
+  val SideTabGroup1: Css         = Css("side-tabs-group-1")
+  val SideTabGroup2: Css         = Css("side-tabs-group-2")
+  val SideTabGroup3: Css         = Css("side-tabs-group-3")
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/ui/ExploreStyles.scala:74]: It seems that group-3 is not used.

Comment (rpiaggio) [reply common/src/main/scala/explore/components/ui/ExploreStyles.scala:74]: And group-1 and -2 are treated the same way, can we unify?

Comment (cquiroz) [reply common/src/main/scala/explore/components/ui/ExploreStyles.scala:74]: You're right we could simplify

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**